src/arch/aarch64/CodeGen.zig

```diff
@@ -4198,10 +4199,20 @@ fn airArg(self: *Self, inst: Air.Inst.Index) !void {
     return self.finishAir(inst, result, .{ .none, .none, .none });
 }
 
+fn airTrap(self: *Self) !void {
+    // Reference: https://github.com/llvm/llvm-project/blob/7ecb5145bae57fce44c53f3e8eacea76078e3825/llvm/lib/Target/AArch64/AArch64FastISel.cpp#L3603-L3613
+    _ = try self.addInst(.{
+        .tag = .brk,
+        .data = .{ .imm16 = 0x1000 },
```

Comment:
In the source you listed, the immediate is actually set to `1` for a trap I believe.

Reply:
You're absolutely right!